src/commands.ts

```diff
@@ -1173,8 +1173,11 @@ export const insertTableFootnote = (
       schema.nodes.inline_footnote
     )
     footnoteIndex =
-      inlineFootnotes.filter(({ pos }) => position + pos <= insertedAt).length +
-      1
+      inlineFootnotes.filter(
+        ({ pos }) =>
+          !(isRejectedInsert(node) || isDeleted(node)) &&
```

Comment:
Shouldn't you keep the deleted?

Reply:
if it's not a rejected delete, we should not count it

Reply:
removed, and will handle it in the delete ticket

Reply:
> if it's not a rejected delete, we should not count it

not rejected delete will stay with a line through it and given that it will stay I think it makes sense to count it.